Methods/Problem11/Program.cs

```diff
@@ -37,7 +37,7 @@ static void Main(string[] args)
                 Console.WriteLine("Please enter option number:");
                 string option = Console.ReadLine();
 
-                if (ValidateOption(option, options))
+                if (IsValidOption(option, options))
```

Comment:
Function's names should be verb like
use explanatory variable in if statement for better readability